### ISSUE TITLE
jo 1.5 thinks it is jo 1.4

### ISSUE THREAD

**Issue by musicinmybrain** [jpmens/jo#163]
For the `jo` 1.5 release, the project version number is still 1.4 in both the Autotools and Meson build systems.

```
$ autoreconf --force --install
$ ./configure 
$ make
$ ./jo -v
jo 1.4
$ meson build
$ ninja -C build
$ ./build/jo -v
jo 1.4
```

**Comment by jpmens** [comment]
If I download `jo-1.5.tar.gz` from the releases page and 

```console
$ tar xf jo-1.5.tar.gz
$ ./configure
$ make
$ ./jo -v
jo 1.5
```

and if I clone

```console
$ git clone git@github.com:jpmens/jo.git
$ cd jo
$ autoreconf -i
$ ./configure
$ make
$ ./jo -v
jo 1.5
```

**Comment by jpmens** [comment]
Ah, I see what you mean. Not sure there's an easy way to fix this ...

**Comment by musicinmybrain** [comment]
It looks like you have already figured out the below, but for any other readers:

It looks like the primary release tarball https://github.com/jpmens/jo/releases/download/1.5/jo-1.5.tar.gz is OK after all, and this is because it doesn’t match the git tag and commit associated with the release.

I was using the automatic tarball for the `1.5` tag, https://github.com/jpmens/jo/archive/1.5/jo-1.5.tar.gz, which corresponds to the “Source Code” archives at https://github.com/jpmens/jo/releases/tag/1.5.

Similarly:

```
$ git clone git@github.com:jpmens/jo.git
$ cd jo
$ git checkout 1.5
$ autoreconf -i
$ ./configure
$ make
$ ./jo -v
jo 1.4
```

**Comment by jpmens** [comment]
Thank you for pointing this out. I think I've been able to fix (i.e. re-tag) using [this incantation](https://stackoverflow.com/questions/8044583/how-can-i-move-a-tag-on-a-git-branch-to-a-different-commit), and using your instructions I now get version `1.5`.

**Comment by musicinmybrain** [comment]
Thanks! That’s probably a reasonable way to handle it, as it has been only hours since the old tag was created and there are likely very few people who might have pulled it. The fix looks good to me.

**Comment by musicinmybrain** [comment]
Reopening this to point out that the new tag is affected by https://github.com/jpmens/jo/issues/164, while the manually-created release tarball seems to be based on https://github.com/jpmens/jo/commit/59e892a2d6011da04caea299780f53742d9e6e7a and is not affected.

**Comment by jpmens** [comment]
I have released [version 1.6](https://github.com/jpmens/jo/releases/tag/1.6), hopefully free of these issues, now. Thanks for your patience and perseverance. :-)

**Comment by musicinmybrain** [comment]
Indeed, version 1.6 looks good to me! Thanks for investigating.
